app/javascript/app/components/ndcs/ndcs-country-accordion/ndcs-country-accordion-actions.js

```diff
@@ -14,13 +14,13 @@ const fetchNdcsCountryAccordionFailed = createAction(
 const fetchNdcsCountryAccordion = createThunkAction(
   'fetchNdcsCountryAccordion',
   params => dispatch => {
-    const { locations, category, compare } = params;
+    const { locations, category, compare, lts } = params;
     if (locations) {
       dispatch(fetchNdcsCountryAccordionInit());
       fetch(
-        `/api/v1/ndcs?location=${locations}&category=${category}${!compare
-          ? '&filter=overview'
-          : ''}`
+        `/api/v1/ndcs?location=${locations}&category=${category}${
+          lts ? '&source=LTS' : ''
```

Comment:
if not LTS we should put the source: `source[]=CAIT&source[]=WB&source[]=NDC Explore`

Reply:
I've merged the PR already, will add it in develop